Natalia_Ivashchenko/Lesson19/pytest19.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import chisla
+from chisla import InputTypeError
+import letters_lower
+import symbols
+
+
+class TestPozitiv:
+    @pytest.mark.parametrize('letters', ['sep', 'oct', 'nov'])
+    def test_lower_letters(self, letters):
+        assert letters_lower.check_letters(letters), True
```

Comment:
после запятой в ассерте пишется сообщение об ошибки, если тест упал, обычно это типа
Test failed because letters is not lower
те осмысленное сообщение, в твоем случае при падении прилетит True, что мало тебе чем поможет
тогда или вообще без сообщения, или с осмысленным сообщением